cross/rhash/PLIST

```diff
@@ -1,12 +1,2 @@
-bin:bin/ed2k-link
```

Comment:
Sure to remove other executables?

Reply:
yes, these are only links to bin/rhash and are created as links upon installation again.